docs/content/latest/explore/query-1-performance/query-tuning-intro.md

```diff
@@ -0,0 +1,380 @@
+    Index Cond: (account_id = 8060)
+        Filter: ((NOT is_paid) AND (updated_at > '2021-04-12 12:00:00'::timestamp without time zone))
+(4 rows)
+
+Time: 46.658 ms
+Previous run time: 147 seconds
+```
+
+### Optimizing `SELECT` by changing table sorting
+
+The following query retrieves data from an account table where some indexes are already defined.
+
+Table definition:
+
+```sql
+yugabyte=# \d accounts
+```
+
+```output
+                                    Table "public.accounts"
+Column        | Type                        | Collation | Nullable | Default
+--------------+-----------------------------+-----------+----------+--------
+id            | bigint                      | For       | not null | nextval('accounts_id_seq'::regclass)
+company_name  | character varying(125)      |           | not null |
+status        | character varying(25)       |           |          |
+first_name    | character varying(55)       |           |          |
+last_name     | character varying(55)       |           |          |
+phone         | character varying(25)       |           |          |
+created_at    | timestamp without time zone |           | not null |
+updated_at    | timestamp without time zone |           | not null |
+product_id    | integer                     |           |          |
+business_type | character varying(55)       |           |          |
+sales_rep     | character varying           |           |          |
+
+Indexes:
+    "accounts_pkey" PRIMARY KEY, lsm (id HASH)
+    "index_on_company_name" lsm (company_name HASH)
+```
+
+`EXPLAIN` output for the query is as follows:
+
+```sql
+yugabyte=# explain SELECT accounts.* FROM accounts 
+    ORDER BY accounts.id desc, accounts.id desc LIMIT 25 OFFSET 0;
+```
+
+```output
+QUERY PLAN
+----------------------------------------------
+Limit (cost=128.22..128.28 rows=25 width=1642)
+    Sort (cost=128.22..130.72 rows=1000 width=1642)
+        Sort Key: id DESC
+            Seq Scan on accounts (cost=0.00..100.00 rows=1000 width=1642)
+```
+
+In this case, a sort is run first, which adds extra time to the query, before running a sequential scan of the table, which, as we have seen, also degrades performance.
+
+To optimize this query, we can adjust the sorting of the table by the primary key to be `DESC` rather than `HASH`.
+
+```sql
+CREATE TABLE public.accounts ( id bigint NOT NULL, ... ,PRIMARY KEY(id desc);
+```
+
+The result is the query no longer does a sequential scan; instead it uses an index scan, cutting execution time from 460ms to 3ms.
+
+```sql
+yugabyte=# explain SELECT accounts.* FROM accounts 
+    ORDER BY accounts.id desc, accounts.id desc LIMIT 25 OFFSET 0;
+```
+
+```output
+QUERY PLAN
+------------------------------------------
+Limit (cost=0.00..2.85 rows=25 width=1642)
+    Index Scan using accounts_pkey on accounts (cost=0.00..114.00 rows=1000 width=1642)
+
+Time: 2.994 ms
+Previous run time: 426.627 ms
+```
+
+### Optimizing `SELECT` using an index
```

Comment:
All of these headings could be imperative. It's a decision we should make across the board sometime, but it works well in this doc.

```suggestion
### Optimize `SELECT` using an index
```